gateway/api/server.go

```diff
@@ -466,4 +467,34 @@ func (api *Api) buildRoutes(route *gin.RouterGroup) {
 		api.Authenticate,
 		api.TrackRequest(analytics.EventUpdateJiraIntegration),
 		apijiraintegration.Put)
+
+	route.POST("/guardrail-rules",
+		api.AllowApiKey,
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		api.TrackRequest(analytics.EventCreateGuardRailRules),
+		apiguardrails.Post)
+
+	route.PUT("/guardrail-rules/:id",
+		api.AllowApiKey,
+		AdminOnlyAccessRole,
+		api.Authenticate,
+		api.TrackRequest(analytics.EventUpdateGuardRailRules),
+		apiguardrails.Put)
+
+	route.GET("/guardrail-rules",
+		api.AllowApiKey,
+		api.Authenticate,
+		apiguardrails.List)
+
+	route.GET("/guardrail-rules/:id",
+		api.AllowApiKey,
+		api.Authenticate,
+		apiguardrails.Get)
+
+	route.DELETE("/guardrail-rules/:id",
+		api.AllowApiKey,
+		api.Authenticate,
+		api.TrackRequest(analytics.EventDeleteGuardRailRules),
+		apiguardrails.Delete)
```

Comment:
for consistency, in frontend and design, we defined the names as "guardrails", so to keep that let's make these endpoints be `/guardrails` instead of `/guardrail-rules`